libs/@guardian/source-react-components/src/icons/Icons.stories.tsx

```diff
@@ -327,19 +331,6 @@ MediumIconsBrandTheme.parameters = {
 		default: 'brandBackground.primary',
 	},
 };
-MediumIconsBrandTheme.decorators = [
```

Comment:
Glad to see these style overrides removed and replaced with a supported method so we're not encouraging people to do the wrong thing 🙈